.pre-commit-config.yaml

```diff
@@ -5,7 +5,7 @@ repos:
       - id: isort
         args: ["--profile", "black", "--line-length", "79"]
 -   repo: https://github.com/psf/black
-    rev: 21.11b1
+    rev: 22.3.0
```

Comment:
Is there a reason for the update ? 

Now you may break the format check because it is not updated in the pipfile

Reply:
I just had some trouble locally with this version, as dependency  click (afair) has changed a bit, so I update it.
I just changed it into the pipfile

Reply:
@Pog3k , we all had. There is some issue between black and the latest click. This make all pre-commit fail.

Reply:
@Mcthomas777 , @sebastianpfischer 
Thx for the info.
Then update it in the pipfile..